src/Core/Datatypes/Tests/DyadicTensorTests.cc

```diff
@@ -318,19 +341,42 @@ TEST(DyadicTensorTest, MultiplyTensorOperator)
 // Coefficient wise multiplication
 TEST(DyadicTensorTest, MultiplyDoubleOperator)
 {
-  Dyadic2DTensor t({Eigen::Vector2d({2, 8}), Eigen::Vector2d({5, 3})});
-  Dyadic2DTensor expected({Eigen::Vector2d({6, 24}), Eigen::Vector2d({15, 9})});
+  Dyadic2DTensor t(Eigen::Vector2d({2, 3}));
+  Dyadic2DTensor expected(Eigen::Vector2d({6, 9}));
 
   Dyadic2DTensor result = t * 3.0;
 
   ASSERT_TRUE(expected == result);
 }
 
+// Coefficient wise multiplication
+// TEST(DyadicTensorTest, DivideTensorOperator)
+// {
+//   Dyadic2DTensor t(Eigen::Vector2d({6, 9}));
+//   Dyadic2DTensor t2(Eigen::Vector2d({4, 3}));
+//   Dyadic2DTensor expected(Eigen::Vector2d({1.5, 3}));
+
+//   Dyadic2DTensor result = t / t2;
+
+//   ASSERT_TRUE(expected == result);
+// }
```

Comment:
I can't get this tensor division operator to work. I made a wrapper to the Eigen::Tensor operator in the same way that I did with tensor multiplication. The Eigen documentation says they both exist, with the caveat that the division operator does not work for integers.

Reply:
Does it not compile, or the test fails at runtime?

Reply:
It does not compile.

Reply:
Here is the single-threaded output of that compiler error: https://pastebin.com/NG966Svs

Reply:
Looks like it's missing an include, since the type is incomplete. You could just include everything in the Unsupported/tensor folder and see if that fixes it, then trim it down to what's needed.

Reply:
@RubioJr9 Do you need this operator right now, or is it just to provide a complete set? I'd rather look at it once this PR is merged, if you don't mind the wait.